client/i18n/schema/trendingSchema.js

```diff
@@ -0,0 +1,13 @@
+import Joi from 'joi';
+
+const schema = Joi.object()
+  .min(60)
+  .max(60)
+  .pattern(/article\d+title/, [Joi.string().required()])
```

Comment:
This is solid and we could possibly make it more bullet proof if we added an iterator for the keys to add the numbers in the keys or hardcode the keys such as the following:
```
Joi.object().keys({
    article0title: Joi.string().required(),
    article0link: Joi.string().uri({ scheme: 'https' }).required(),
...
})
```

Reply:
Hey @ahmadabdolsaheb, sure, we could do that. I don't see the harm in having a really plain, easy to read schema for something like this. I seriously doubt we'll be updating the schema anytime soon.

I'll go in and revert to hardcoded keys / values for the schema validation.